browser/ui/webui/brave_webui_source.cc

```diff
@@ -784,7 +784,6 @@ void CustomizeWebUIHTMLSource(const std::string &name,
         { "rewardsPanelOffText1", IDS_BRAVE_UI_REWARDS_PANEL_OFF_TEXT1 },
         { "rewardsPanelOffText2", IDS_BRAVE_UI_REWARDS_PANEL_OFF_TEXT2 },
         { "rewardsPanelText1", IDS_BRAVE_UI_REWARDS_PANEL_TEXT1 },
-        { "rewardsPanelText2", IDS_BRAVE_UI_REWARDS_PANEL_TEXT2 },
         { "rewardsPanelText3", IDS_BRAVE_UI_REWARDS_PANEL_TEXT3 },
```

Comment:
Nit: `IDS_BRAVE_UI_REWARDS_PANEL_TEXT3` `==>` `IDS_BRAVE_UI_REWARDS_PANEL_TEXT2`, `IDS_BRAVE_UI_REWARDS_PANEL_TEXT4` `==>` `IDS_BRAVE_UI_REWARDS_PANEL_TEXT3`

Reply:
Unfortunately I think that will re-trigger translations for the changed keys, which we might want to avoid for the PR (which will be uplifted). Let's just be sure not to use `...N` names in the future.